src/ResourceManagement/Resource/Microsoft.Azure.Management.ResourceManager/generate.cmd

```diff
@@ -9,7 +9,7 @@ set autoRestVersion=0.17.0-Nightly20160629
 set source=-Source https://www.myget.org/F/autorest/api/v2
 
 set resSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-resources/resources/2016-09-01/swagger/resources.json"
-set lockSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-resources/locks/2015-01-01/swagger/locks.json"
+set lockSpecFile="https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-resources/locks/2016-09-01/swagger/locks.json"
```

Comment:
I was using a private package provided by Garrett. He has some fixes there. I think I can update the version later when he has all the fixes checked in.